### PR TITLE
[NS Records] Create gochiosu

### PR DESCRIPTION
**Pull request by YukiVN**
## Requirements
Unless explicitly specified otherwise by a **maintainer** or in the requirement description, your domain must pass **ALL** the indicated requirements above.

Please note that we reserve the rights not to accept any domain at our own discretion.

- [ ] The file is in the `domains` folder and is in the JSON format.
- [ ] The file's name is all lowercased and alphanumeric. 
- [ ] You have completed your website. 
- [ ] The website is reachable.  
- [ ] The CNAME record doesn't contain `https://` or `/`.  
- [ ] There is sufficient information at the `owner` field.

### CODE REVIEW

File: domains/gochiosu
Comment (CuteDog5695) [inline domains/gochiosu:1]: Domain needs to be in json format. Also the file is blank?

**Review by iostpa — CHANGES_REQUESTED** [review]

**Review by iostpa — CHANGES_REQUESTED** [review]
You need owner field.
Please read the docs on the file format.

File: domains/gochiosu.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+  "owner": {
+    "username": "YukiVN"
```
Comment (CuteDog5695) [inline domains/gochiosu.json:3]:
```suggestion
    "username": "YukiVN",
```

**Review by iostpa — DISMISSED** [review]

**Comment by CuteDog5695** [comment]
Seems fine now, wait for the admin

**Comment by CuteDog5695** [comment]
> Seems fine now, wait for the admin

In the mean time, please state why you want NS records.

**Comment by YukiVN** [comment]
> > Seems fine now, wait for the admin
> 
> In the mean time, please state why you want NS records.

I want to try open kind of server like web server to explore and study
I need a domain with NS record to hide my public home IP behind Cloudflare Proxy, to be safe from bad guys.
It is much easier for me to manage my domain DNS via Cloudflare Dashboard.
I confirm I will not use the domain for any illegal/inappropriate/NSFW activities, nor give access to another person. If this happens, my domain will be suspended.

**Comment by YukiVN** [comment]
> > > Seems fine now, wait for the admin
> > 
> > 
> > In the mean time, please state why you want NS records.
> 
> I want to try open kind of server like web server to explore and study I need a domain with NS record to hide my public home IP behind Cloudflare Proxy, to be safe from bad guys. It is much easier for me to manage my domain DNS via Cloudflare Dashboard. 
Btw i want Cloudflare certs too
I confirm I will not use the domain for any illegal/inappropriate/NSFW activities, nor give access to another person. If this happens, my domain will be suspended.

**Comment by CuteDog5695** [comment]
Edited comment to not be in a quote, but here are my opinions.

For the home IP part, thats a valid reason. For the dashboard thing, pretty much not a valid reason because 5000+ users can edit their domain via github. And for the cloudflare cert, you can always use certbot.
In general, you might be accepted because of the home IP part.

**Comment by YukiVN** [comment]
> Edited comment to not be in a quote, but here are my opinions.
> 
> For the home IP part, thats a valid reason. For the dashboard thing, pretty much not a valid reason because 5000+ users can edit their domain via github. And for the cloudflare cert, you can always use certbot. In general, you might be accepted because of the home IP part.

Will this pull request can be merged? 😭

**Comment by CuteDog5695** [comment]
Depends on the admin.

**Comment by YukiVN** [comment]
> Depends on the admin.

Tysm

**Comment by Tian-MC** [comment]
I wanted to talk to the admin because YukiVN is a friend of mine.

We are trying to create this osu server with the domain is-a.dev.
The problem is that we have to use Cloudflare Nameservers because we need proxies, SSL certificates, records and all the other stuff that only work with cloudflare (for some reason).
I had already tried using certbot and other ways to use LetsEncrypt before, but it only works with cloudflare certs.
Then we should use the certificate origin in strict mode.

**Comment by wdhdev** [comment]
> I had already tried using certbot and other ways to use LetsEncrypt before

They should work, there isn't anything as to why they aren't. It might be a configuration issue on your end.

**Comment by Tian-MC** [comment]
> They should work, there isn't anything as to why they aren't. It might be a configuration issue on your end.

I already try as I said, it doesn't work because it need custom SSL to "Full (Strict)", and certbot are not validated

**Comment by wdhdev** [comment]
By default we have it set to full if you enable CF proxy. Changing it from full to strict doesn't change anything on how you can request an SSL. Try generate an SSL using a DNS challenge instead.

**Comment by Tian-MC** [comment]
Yea, i know, but we need Full STRICT, not just full.
Idk why osu server works just with full strict.
It's working also with flexible, but is not safe for an osu server that's make a lot of requests

**Pull request closed without merging**